packages/datatrak-web/src/views/WelcomeScreens/Carousel.tsx

```diff
@@ -0,0 +1,70 @@
+/*
+ * Tupaia
+ *  Copyright (c) 2017 - 2024 Beyond Essential Systems Pty Ltd
+ */
+
+import React from 'react';
+import SwipeableViews from 'react-swipeable-views';
+import styled from 'styled-components';
+import { Typography } from '@material-ui/core';
+import { CarouselDots } from './CarouselDots';
+
+const Inner = styled.div`
+  height: 100%;
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-end;
+`;
+
+const ImageContainer = styled.div`
+  flex: 1;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  padding: 0 0.5rem;
+  inline-size: 100%;
+  max-inline-size: 100%;
+`;
+
+const Title = styled(Typography).attrs({
+  variant: 'h1',
+})`
+  font-size: 1rem;
+  margin-bottom: 1rem;
```

Comment:
```suggestion
  margin-block-end: 1rem;
```

Reply:
I’d also suggest [`text-wrap: balance`](https://developer.mozilla.org/en-US/docs/Web/CSS/text-wrap) on `<Title>` and `<Text>`